tabbycat/participants/emoji.py

```diff
@@ -24,12 +24,14 @@ def set_emoji(teams, tournament):
         team.save()
 
 
-def pick_unused_emoji():
+def pick_unused_emoji(tournament_id=None):
```

Comment:
Could use the entire `tournament` object as a parameter rather than just `id`.

Reply:
The idea was that you don't need an additional query if all you have is the ID (if I remember my Django 101 right, if you have a `Team` object, `team.tournament` would run a query, but `team.tournament_id` would not).   
This, however, doesn't currently happen anywhere in the code and is rather unlikely to in the future, so I can of course change it

Reply:
This was a minor point, so I merged regardless. Using model objects in queries is done smarter; it realizes it doesn't need to join the `tournaments_tournament` table for it's primary keys.